### ISSUE TITLE
Cannot retrieve the occurences of event 

### ISSUE THREAD

**Issue by jdewilde** [Informatievlaanderen/GIPOD#144]
It's not possible to retrieve all the occurences of the event with GipodId 3706553. Maybe you should limit the occurences of an public domain occupancy.

Even the UI https://new.beta.gipod.vlaanderen.be/innames/3706553 seems to load forever.

**Comment by bertzor** [comment]
Hi @jdewilde,

Thank you for reporting this issue. I confirmed the bug and we will resolve the issue.

Kind regards,

Bert

**Comment by olivermay** [comment]
We did some further investigation on this specific issue, the conclusion is that this is mainly a data issue. Specifically it's not the amount of occurences, but actually the amount of exceptions that causes the UI to freeze when loading. The event is a yearly event, but was entered as a daily event with all days except one for every year added as an exception, instead of a yearly event, resulting in an exception list of ~82K records. 

That said, we will add an issue to the backlog, to do a sanity check on the data of occurences entered trough the API, as you suggested. 

I also reported this as a bug for the UI, with a low priority for now, as it's only affected by one intake (that we know of).

@jdewilde is this issue blocking anything at your side? If this is the case I would suggest that we contact Ghent and ask them if we can remove, or change this specific event as a quick fix.
